character-controller.js

```diff
@@ -1136,38 +1136,39 @@ class LocalPlayer extends UninterpolatedPlayer {
     });
   }
   grab(app, hand = 'left') {
-    let position   = null,
-        quaternion = null;
+    if(this instanceof LocalPlayer) {

```

Comment:
- Can player call grab when it's not local player?
- Even though if it's the case, you can check this with different code. Please reference how it did on existing code.